packages/rocketchat-apps/server/converters/users.js

```diff
@@ -50,7 +50,7 @@ export class AppUsersConverter {
 			case 'bot':
 				return UserType.BOT;
 			default:
-				throw new Error('Unknown user type of:', type);
+				return type.toUpperCase();
```

Comment:
Should we keep spitting out at least a warning?  

To at least make sure we are always aware that we have types that aren't valid?

We should be striving for default to never be hit.  If we have to do so by being annoying to our self maybe not such a bad thing?

Reply:
Added some warning logs now. 👍